modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/product_options/ProductOptionRadio.js

```diff
@@ -87,8 +87,10 @@ const ProductOptionRadio = ({
 							quantity: defaultProductOptionValue?.quantity,
 							skuId: defaultProductOptionValue?.skuId,
 							skuOptionKey: productOption.key,
+							skuOptionName: productOption.name,
 							skuOptionValueKey: defaultProductOptionValue?.key,
-							value: [defaultProductOptionValue?.key],
+							skuOptionValueName: defaultProductOptionValue?.name,
+							value: defaultProductOptionValue?.key,
```

Comment:
Same here: why not array? My thought is that is the service looking for arrays? Is it possible?

Reply:
The service is not looking for arrays. In the JS, these implementations are strictly for front-end purposes.

Reply:
Please correct me here if I am wrong and I will re-fix accordingly.